CPAC/pipeline/engine.py

```diff
@@ -2043,9 +2045,30 @@ def ingress_freesurfer(wf, rpool, cfg, data_paths, unique_id, part_id, ses_id):
                 creds_path=data_paths["creds_path"],
                 dl_dir=cfg.pipeline_setup["working_directory"]["path"],
             )
-            rpool.set_data(
-                key, fs_ingress, "outputspec.data", {}, "", f"fs_{key}_ingress"
-            )
+            # if .mgz reorient to RPI
```

Comment:
```suggestion
            # reorient *.mgz
```

not just to RPI, to whatever's in the config, right?